tests/chainer_tests/links_tests/model_tests/test_classifier.py

```diff
@@ -0,0 +1,83 @@
+import unittest
+
+import mock
+import numpy
+
+import chainer
+from chainer import links
+from chainer import testing
+from chainer.testing import attr
+
+
+@testing.parameterize(*testing.product({
+    'compute_accuracy': [True, False],
+    'x_num': [1, 2]
+}))
+class TestClassifier(unittest.TestCase):
+
+    def setUp(self):
+        self.link = links.Classifier(links.Linear(10, 3))
```

Comment:
Either will do. But I thought plain `Link` class is not expected to instantiate as I usually use this class as an  interface class of concrete `Link` objects, although the instantiation of `Link` class is not prohibited.